### PR TITLE
Mh tabbed interface

### PR DESCRIPTION
**Pull request by merrillhuang**
Merge tabbed interface branch into main.

### CODE REVIEW

**Review by jessjno — APPROVED** [review]
Looks good to me

**Comment by armstrca** [comment]
Please merge all prior pull requests to main for more efficient code review

**Comment by merrillhuang** [comment]
Will do! Waiting on peer approval for my user profile pull request.

**Comment by merrillhuang** [comment]
All changes from prior branches merged into this branch.

**Review by armstrca — APPROVED** [review]
Slick use of partials!

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,33 @@
 class UsersController < ApplicationController
+  before_action :get_user, only: [:show, :liked, :feed, :discover, :followers, :following]
```
Comment (Ritumka) [inline app/controllers/users_controller.rb:2]: I like how you added 'before_action' here!

File: app/views/users/_connections.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="row">
+  <div class="col-md-6 offset-3">
+    <div class="list-group">
+      <% connections.each do |connection| %>
+        <%= link_to "#{connection.username}", user_path(connection.username), class: "list-group-item list-group-item-action" %>
+      <% end %>
+    </div>
+  </div>
+</div>
```
Comment (Ritumka) [inline app/views/users/_connections.html.erb:9]: That's interesting how you make this '_connection.html.erb' partial work both for followers and for following.

File: app/views/users/_tabs.html.erb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Followers', followers_path(user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Following', following_path(user.username), class: "nav-link" %>
+      </li>
+    </ul>
+  </div>
+</div>
```
Comment (Ritumka) [inline app/views/users/_tabs.html.erb:28]: Nice job with making all this links as partial!

**Review by Ritumka — COMMENTED** [review]
Good job!

**Pull request merged**